### DISCUSSION TITLE
AttributeError: 'int' object has no attribute 'items' when loading previously retrained model

### DISCUSSION THREAD

**Discussion by traceymai** [explosion/spaCy#7213] [Help: Coding & Implementations]
Hi, I've been trying to do `spacy.load()` with this retrained `en_core_web_trf` model but keeps getting this error. Any idea what might be happening here?
![image](https://user-images.githubusercontent.com/75960921/109226274-53336e00-7823-11eb-9148-83c231e1b9ab.png)

**Comment by svlandeg** [comment]
Hi! It's a little difficult to guess what might be going on without a reproducible code snippet. Could you paste the code and/or explain the steps that led to this error? Could you also share the installed versions of `spacy`, `spacy-transformers`, `thinc` and `en_core_web_trf`?
